pkgs/development/beam-modules/build-mix.nix

```diff
@@ -37,7 +38,8 @@ let
       addToSearchPath ERL_LIBS "$1/lib/erlang/lib"
     '';
 
-    nativeBuildInputs = [ elixir hex ];
+    buildInputs = buildInputs ++ [ ];
```

Comment:
do we need to change the buildInputs part then ?

Reply:
We don't need it _yet_, but if someone wants to add buildInputs, this will prevent them from making the same mistake again.

Reply:
I see, thanks!